README.md

```diff
@@ -1,14 +1,17 @@
 # Getting Started with the Pub/Sub API
 
-- [About](#about)
-- [Resources](#resources)
+- [About Pub/Sub API](#about-pub/sub-api)
```

Comment:
I believe this should be `[About Pub/Sub API](#about-pubsub-api)` instead

Reply:
Fixed